backend/src/apiserver/template/v2_template.go

```diff
@@ -248,20 +248,20 @@ func (t *V2Spec) ParametersJSON() (string, error) {
 func (t *V2Spec) RunWorkflow(modelRun *model.Run, options RunWorkflowOptions) (util.ExecutionSpec, error) {
 	bytes, err := protojson.Marshal(t.spec)
 	if err != nil {
-		return nil, util.Wrap(err, "Failed marshal pipeline spec to json")
+		return nil, util.NewInternalServerError(err, "Failed to marshal pipeline spec to json")
 	}
 	spec := &structpb.Struct{}
 	if err := protojson.Unmarshal(bytes, spec); err != nil {
-		return nil, util.Wrap(err, "Failed to parse pipeline spec")
+		return nil, util.NewInternalServerError(err, "Failed to parse pipeline spec")
 	}
 	job := &pipelinespec.PipelineJob{PipelineSpec: spec}
 	jobRuntimeConfig, err := modelToPipelineJobRuntimeConfig(&modelRun.RuntimeConfig)
 	if err != nil {
-		return nil, util.Wrap(err, "Failed to convert to PipelineJob RuntimeConfig")
+		return nil, util.NewInternalServerError(err, "Failed to convert to PipelineJob RuntimeConfig")
```

Comment:
Should we change it to `Failed to convert PipelineJob RuntimeConfig`?

Reply:
Here we are converting RuntimeConfig from model type into pipeline job format, so I think the error message is accurate.